### PR TITLE
Feature/create items

### PR DESCRIPTION
**Pull request by rashmoni**
Completed Create item including image file save a blob.

### CODE REVIEW

File: src/main/java/com/novare/tredara/controllers/ItemController.java
Comment (AlirezaKToosi) [inline src/main/java/com/novare/tredara/controllers/ItemController.java:1]: Do we need to send User Id in login Response?

Comment (OlgaPinchuk) [reply src/main/java/com/novare/tredara/controllers/ItemController.java:1]: Yes

Comment (rashmoni) [reply src/main/java/com/novare/tredara/controllers/ItemController.java:1]: Sorry did not get you question. Please elaborate.

**Review by AlirezaKToosi — APPROVED** [review]
Looks good  but i have already posted a comment for ItemDTO

**Comment by rashmoni** [comment]
> Looks good but i have already posted a comment for ItemDTO

Do you mean ItemController?

File: src/main/java/com/novare/tredara/controllers/ItemController.java
Comment (AlirezaKToosi) [reply src/main/java/com/novare/tredara/controllers/ItemController.java:1]: I understand and my problem is solved. ،Thanks

**Review by aggarrohit — APPROVED** [review]

**Pull request merged**